### ISSUE TITLE
Reduce/disable server loggng on startup

### ISSUE THREAD

**Issue by purejosh** [VazkiiMods/Zeta#66]
The output of logging from quark is crazy. Can you please add an option to either turn off or reduce this verbosity? You really don't need a line for every module. 

```
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Discovered 163 modules to load.
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Abacus...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Adjustable Chat...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Ambient Discs...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Ancient Tomes...
[11:37:13] [modloading-worker-0/INFO] [de.to.pa.PacketFixer/]: Packet Fixer has been initialized successfully
[11:37:13] [modloading-worker-0/WARN] [de.ke.lo.LoadMyResources/]: ## WARNING ## 'Load My Resources' is a client mod and has no effect when loaded on a server!
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Ancient Wood...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Armed Armor Stands...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Auto Walk Keybind...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Automatic Recipe Unlock...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Automatic Tool Restock...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Azalea Wood...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Back Button Keybind...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Backpack...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Beacon Redirection...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Better Elytra Rocket...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Big Stone Clusters...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Blossom Trees...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Bottled Cloud...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Buckets Show Inhabitants...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Camera...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Campfires Boost Elytra...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Celebratory Lamps...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Chains Connect Blocks...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Chest Searching...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Chorus Vegetation...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Chute...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Climate Control Remover...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Color Runes...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Compasses Work Everywhere...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Compressed Blocks...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Coral On Cactus...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Corundum...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Crabs...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Crafter...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Crate...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Diamond Repair...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Dispensers Place Blocks...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Double Door Opening...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Dragon Scales...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Duskbound Blocks...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Dyeable Item Frames...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Easy Transfering...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Elytra Indicator...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Emotes...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Enchantment Predicates...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Enchantments Begone...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Ender Watcher...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Endermosh Music Disc...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Enhanced Ladders...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Expanded Item Interactions...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Fairy Rings...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Fallen Logs...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Feeding Trough...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Forgotten...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Foxhound...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Framed Glass...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Game Nerfs...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Glass Item Frame...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Glass Shard...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Glimmering Weald...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Gold Bars...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Gold Tools Have Fortune...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Grab Chickens...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Grate...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Gravisand...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Greener Grass...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Hedges...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Hoe Harvesting...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Hollow Logs...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Horses Swim...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Hotbar Changer...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Improved Sponges...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Improved Tooltips...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Industrial Palette...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Inventory Sorting...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Iron Rod...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Item Sharing...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Japanese Palette...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Leaf Carpet...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Lock Rotation...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Long Range Pick Block...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Magma Keeps Concrete Powder...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Magnets...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Map Washing...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Matrix Enchanting...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Metal Buttons...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Microcrafting Helper...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Midori...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Monster Box...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module More Banner Layers...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module More Brick Types...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module More Mud Blocks...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module More Note Block Sounds...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module More Potted Plants...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Narrator Readout...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Nether Brick Fence Gate...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Nether Obsidian Spikes...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module New Stone Types...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module No Durability On Cosmetics...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module No More Lava Pockets...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Obsidian Plate...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Overlay Shader...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Parrot Eggs...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Pat The Dogs...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Pathfinder Maps...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Permafrost...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Petals On Water...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Pickarang...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Pig Litters...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Pipes...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Pistons Move Tile Entities...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Poison Potato Usage...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Quick Armor Swapping...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Rainbow Lamps...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Raw Metal Bricks...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Reacharound Placing...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Redstone Randomizer...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Renewable Spore Blossoms...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Replace Scaffolding...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Rope...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Safer Creatures...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Seed Pouch...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Shear Vines...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Shiba...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Shingles...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Shulker Packing...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Simple Harvest...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Skull Pikes...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Slabs To Blocks...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Slime In A Bucket...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Slimes To Magma Cubes...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Snow Golem Player Heads...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Soul Candles...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Soul Sandstone...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Spawner Replacer...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Spiral Spires...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Stonelings...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Stools...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Sturdy Stone...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Thatch...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Tiny Potato...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Torch Arrow...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Toretoise...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Totem Of Holding...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Trowel...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Usage Ticker...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Uses For Curses...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Utility Recipes...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Variant Animal Textures...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Variant Bookshelves...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Variant Chests...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Variant Furnaces...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Variant Ladders...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Variant Selector...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Vertical Planks...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Vertical Slabs...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Vexes Die With Their Masters...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Villager Rerolling Rework...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Villagers Follow Emeralds...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Wooden Posts...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Wool Shuts Up Minecarts...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Wraith...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module Zombie Villagers On Normal...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructing module More Stone Variants...
[11:37:13] [modloading-worker-0/INFO] [quark-zeta/]: Constructed 163 modules.
```

**Comment by quat1024** [comment]
It is useful for debugging loading failures (if it crashes initializing a module, you can look at the last printout to see which one crashed) but I agree that it's a bit wordy. I'm not sure if this stuff happens before or after Zeta loads its own config file. Mayeb there are other ways to improve the debugging (like catching and re-throwing exceptions to include the module name)

**Comment by purejosh** [comment]
Do these loglines only print if there's a compatible mod installed? Maybe a `debug: true` entry in the config would be useful to reduce these lines.
